index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ember.js</title>
+  </head>
+  <body>
+    <script type="module">
+      import 'qunit/qunit/qunit.js';
+      import 'qunit/qunit/qunit.css';
+
+      window.EmberENV = {};
+
+      // Test under octane defaults
+      EmberENV._DEFAULT_ASYNC_OBSERVERS = true;
+
+      // Test for "hooks in EmberENV.EMBER_LOAD_HOOKS['hookName'] get executed"
+      EmberENV.EMBER_LOAD_HOOKS = EmberENV.EMBER_LOAD_HOOKS || {};
+      EmberENV.EMBER_LOAD_HOOKS.__before_ember_test_hook__ =
+        EmberENV.EMBER_LOAD_HOOKS.__before_ember_test_hook__ || [];
+      EmberENV.__test_hook_count__ = 0;
+      EmberENV.EMBER_LOAD_HOOKS.__before_ember_test_hook__.push(function (object) {
+        EmberENV.__test_hook_count__ += object;
+      });
+
+      // Handle extending prototypes
+      EmberENV['EXTEND_PROTOTYPES'] = !!QUnit.urlParams.EXTEND_PROTOTYPES;
+
+      // Handle testing feature flags
+      if (QUnit.urlParams.ENABLE_OPTIONAL_FEATURES) {
+        EmberENV.ENABLE_OPTIONAL_FEATURES = true;
+      }
+
+      EmberENV['RAISE_ON_DEPRECATION'] = true;
+
+      if (QUnit.urlParams.ALL_DEPRECATIONS_ENABLED) {
+          EmberENV['_ALL_DEPRECATIONS_ENABLED'] = QUnit.urlParams.ALL_DEPRECATIONS_ENABLED;
```

Comment:
```suggestion
          EmberENV['_ALL_DEPRECATIONS_ENABLED'] = !!QUnit.urlParams.ALL_DEPRECATIONS_ENABLED;
```
nearly got through again

Reply:
Oof, thanks. This was missed because the other PR fixing it isn't merged yet. If it was I would have seen the conflict.

Reply:
ah yeah, if it got caught as a move -- I need to follow up on that